<launch>
    <node pkg="avc" type="image_transform" name="image_transform" output="screen">
        <param name="transform_file" value="$(find avc)/launch/transform.yaml"/>
    </node>
</launch>